cogs/remind_me.py

```diff
@@ -27,14 +28,14 @@
     from collections.abc import Iterator
 
 
-logger: Logger = logging.getLogger("TeX-Bot")
+logger: Final[Logger] = logging.getLogger("TeX-Bot")
 
 
 class RemindMeCommandCog(TeXBotBaseCog):
-    """Cog class that defines the "/remindme" command and its call-back method."""
+    """Cog class that defines the "/remind-me" command and its call-back method."""
 
     @staticmethod
-    async def autocomplete_get_delays(ctx: TeXBotAutocompleteContext) -> set[str]:  # noqa: C901, PLR0912, PLR0915
+    async def autocomplete_get_delays(ctx: TeXBotAutocompleteContext) -> Set[discord.OptionChoice] | Set[str]:  # noqa: C901,PLR0912,PLR0915,E501
```

Comment:
Although there is no formal rule for how to annotate multiple `# noqa` comments in a list, all the examples I have seen from ruff & mypy's documentation suggest to just separate with commas and not spaces to reduce line length. Especially when they're more focused for machine reading rather than developer reading.

Reply:
hmmm, I do think in general it's easier to read if they have the space so I think i'd rather that, but in any case it definitely needs to be consistent so if you wanted to do it without the spaces you'd need to go round remove them all

Reply:
Cool, I'll remove those changes from this PR and we can formalise on having spaces.

I'm planning on adding rules like this to a custom linter based on a flake8 plugin. We'll use ruff for the common linting rules because it's written in rust so is much faster and the ones that aren't handled by that (like command names being hyphenated etc) can be implemented in my custom linter. (This will be a separate repo project)